core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1316,6 +1316,46 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_EMBEDDED_JOURNAL_NETWORK_REQUEST_TIMEOUT_MS =
+      new Builder(Name.MASTER_EMBEDDED_JOURNAL_NETWORK_REQUEST_TIMEOUT_MS)
+          .setDefaultValue("5sec")
+          .setDescription("Timeout for communication between embedded journal masters.")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
+          .setScope(Scope.MASTER)
+          .build();
+  public static final PropertyKey MASTER_EMBEDDED_JOURNAL_NETWORK_FLOWCONTROL_WINDOW =
+      new Builder(Name.MASTER_EMBEDDED_JOURNAL_NETWORK_FLOWCONTROL_WINDOW)
+          .setDefaultValue("2MB")
+          .setDescription("The HTTP2 flow control window used by copycat connections. Larger "
+              + "value will allow more data to be buffered but will use more memory.")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
+          .setScope(Scope.MASTER)
+          .build();
+  public static final PropertyKey MASTER_EMBEDDED_JOURNAL_NETWORK_KEEPALIVE_TIME_MS =
+      new Builder(Name.MASTER_EMBEDDED_JOURNAL_NETWORK_KEEPALIVE_TIME_MS)
+          .setDefaultValue(Long.MAX_VALUE)
```

Comment:
Why set the default value to be that big? Will it cause some copycat client connections which should be timeout but keep waiting and use out resources like open file limits?

Reply:
Nice catch. I needed to define server side properties and enforce ping traffic from the server.
However, copycat's request timeout (MASTER_EMBEDDED_JOURNAL_TRANSPORT_REQUEST_TIMEOUT_MS) renders network level optimizations useless so I removed all network related props.